kvm.yaml

```diff
@@ -69,6 +106,23 @@
       version: 0.2.0
     - name: kvm-operator
       version: 3.10.0
+- version: 9.3.0
```

Comment:
9.3.0. there is no 9.1.0 and 9.2.0 but I recall @MarcelMue that you talked about 9.3.0. Am I wrong?

Reply:
I might have confused `kvm` and `aws` - `9.1.0` looks correct for kvm.